docs/python_api.rst

```diff
@@ -0,0 +1,398 @@
+|               |                                                     |
+|               | ``c.sites(1).networks('10.0.0.0/24').patch({...})`` |
++---------------+-----------------------------------------------------+
+
+As opposed to PUT which can replace existing data, PATCH is "safer" in that
+regard. If you don't provide some keys in your update, they will be untouched.
+
+As with PUT and POST, a successful one should return the new payload.
+
+Like Creating, PATCH also supports BULK operations.
+
+.. code-block:: python
+
+   net = c.sites(1).networks('10.0.1.0/24').get()
+   # {u'attributes': {u'dc': u'sfo'},
+   # u'id': 3,
+   # u'ip_version': u'4',
+   # u'is_ip': False,
+   # u'network_address': u'10.0.1.0',
+   # u'parent_id': 1,
+   # u'prefix_length': 24,
+   # u'site_id': 1,
+   # u'state': u'allocated'}
+
+   net.pop('attributes')
+   c.sites(1).networks('10.0.1.0/24').patch(net)
+   # {u'attributes': {u'dc': u'sfo'},
+   #  u'id': 3,
+   #  u'ip_version': u'4',
+   #  u'is_ip': False,
+   #  u'network_address': u'10.0.1.0',
+   #  u'parent_id': 1,
+   #  u'prefix_length': 24,
+   #  u'site_id': 1,
+   #  u'state': u'allocated'}
+
+Deleting Resources
+------------------
+
++---------------+-------------------------------------------------+
+| HTTP Method   | DELETE                                          |
++---------------+-------------------------------------------------+
+| HTTP Path     | ``DELETE /api/sites/1/networks/1/``             |
+|               |                                                 |
+|               | ``DELETE /api/sites/1/networks/10.0.0.0/24/``   |
++---------------+-------------------------------------------------+
+| Python Client | ``c.sites(1).networks(1).delete()``             |
+|               |                                                 |
+|               | ``c.sites(1).networks('10.0.0.0/24').delete()`` |
++---------------+-------------------------------------------------+
+
+This one should be straight forward and there is no payload to deal with. Will
+return bool.
+
+
+.. code-block:: python
+
+   c.sites(1).networks('10.0.1.0/24').delete()
+   # True
+
+Querying by Attribute Values
+----------------------------
+
++---------------+-------------------------------------------------------------+
+| HTTP Method   | GET                                                         |
++---------------+-------------------------------------------------------------+
+| HTTP Path     | ``GET /api/sites/1/networks/query/?query='set query here'`` |
++---------------+-------------------------------------------------------------+
+| Python Client | ``c.sites(1).networks.query.get(query='set query here')``   |
++---------------+-------------------------------------------------------------+
+
+Set queries are the way to filter based an attributes and their values. The
+syntax is typical set query syntax and is lightly discussed here:
+:ref:`set_query_ref`
```

Comment:
This isn't rendering properly for some reason. Since this is also an inline ref which should render to a link, you could just make it:

 ```
and is lightly discussed in :ref:`set_queries`.
```